Penteract/Assets/Scripts/Enemy.h

```diff
@@ -0,0 +1,32 @@
+#pragma once
+
+#include "Character.h"
+
+class Enemy : public Character {
+public:
+	// ------- Contructors ------- //
+	Enemy() {};
+
+	Enemy(int lifePoints_, float movementSpeed_, int damageHit_, int fallingSpeed_, float searchRadius_, float meleeRange_, float timeToDie_)
+		: fallingSpeed(fallingSpeed_)
+		, searchRadius(searchRadius_) 
+		, meleeRange(meleeRange_)
+		, timeToDie(timeToDie_) {
+		lifePoints = lifePoints_;
+		movementSpeed = movementSpeed_;
+		damageHit = damageHit_;
+		SetTotalLifePoints(lifePoints);
+	}
+
+	// ------- Core Functions ------ //
+	void SetFallingSpeed(int fallingSpeed);
+	void SetSearchRadius(float searchRadius);
+	void SetMeleeRange(float meleeRange);
+	void SetTimeToDie(float timeToDie);
```

Comment:
Missing _ at the end. Incoherence between header and source files

Reply:
Perfect, I will fix it in the next commit ;) Thanks!